src/snmpd/Makefile

```diff
@@ -19,8 +19,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	rm -rf net-snmp-$(SNMPD_VERSION)
 
 	# download debian net-snmp
-	# TODO: backup source packages to Azure Blob
-	dget http://deb.debian.org/debian/pool/main/n/net-snmp/net-snmp_$(SNMPD_VERSION_FULL).dsc
+	dget -u https://sonicstorage.blob.core.windows.net/debian/pool/main/n/net-snmp/net-snmp_$(SNMPD_VERSION_FULL).dsc
```

Comment:
I guess the .dsc file is point to other files on http://deb.debian.org/. We need to cache all 3 files and `dpkg-source -x` as before.

Reply:
all cached. 

check out here

```
lgh@gulv-vm3:~$ dget -u https://sonicstorage.blob.core.windows.net/debian/pool/main/n/net-snmp/net-snmp_5.7.3+dfsg-5.dsc
dget: retrieving https://sonicstorage.blob.core.windows.net/debian/pool/main/n/net-snmp/net-snmp_5.7.3+dfsg-5.dsc
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2968  100  2968    0     0  10450      0 --:--:-- --:--:-- --:--:-- 10414
dget: retrieving https://sonicstorage.blob.core.windows.net/debian/pool/main/n/net-snmp/net-snmp_5.7.3+dfsg.orig.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 3292k  100 3292k    0     0  3117k      0  0:00:01  0:00:01 --:--:-- 3117k
dget: using existing net-snmp_5.7.3+dfsg-5.debian.tar.xz
```